Source/Meadow.Foundation.Core.Samples/Leds.PwmLed_Onboard_Sample/MeadowApp.cs

```diff
@@ -24,56 +26,58 @@ public MeadowApp()
         public void ConfigurePeripherals()
         {
             Console.WriteLine("Creating peripherals...");
-            this._redPwmLed = new PwmLed(Device,
-                Device.Pins.OnboardLedRed, TypicalForwardVoltage.ResistorLimited, CircuitTerminationType.High);
-            this._greenPwmLed = new PwmLed(Device,
-                Device.Pins.OnboardLedGreen, TypicalForwardVoltage.ResistorLimited, CircuitTerminationType.High);
-            this._bluePwmLed = new PwmLed(Device,
-                Device.Pins.OnboardLedBlue, TypicalForwardVoltage.ResistorLimited, CircuitTerminationType.High);
+            redPwmLed = new PwmLed(Device, Device.Pins.OnboardLedRed, TypicalForwardVoltage.ResistorLimited, CircuitTerminationType.High);
```

Comment:
I tend to put a parameter per line if its too long. Oherwise, the instruction might wrap akwardly on the DocFX API code block region.